table.csv

```diff
@@ -440,6 +440,10 @@ zcash-tx,             Zcash Tx,                                         0xc1
 stellar-block,        Stellar Block,                                    0xd0
 stellar-tx,           Stellar Tx,                                       0xd1
 
+leofcoin-block,       Leofcoin Block,                                   0x0A
+leofcoin-tx,          Leofcoin Transaction,                             0x54
+leofcoin-pr,          Leofcoin Peer Reputation,                         0x52
```

Comment:
Cool! Is there an IPLD plugin too?

Reply:
@diasdavid Almost, I discovered today that there is a interface-ipld-format repo so going to go through the readme there first before pushing.